packages/charts/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -142,14 +143,22 @@ export class CartesianChartBase
   public componentDidMount(): void {
     this._fitParentContainer();
     if (
-      this.props.chartType === ChartTypes.HorizontalBarChartWithAxis &&
+      [ChartTypes.HorizontalBarChartWithAxis, ChartTypes.HeatMapChart].includes(this.props.chartType) &&
```

Comment:
define this array once as its being used at multiple places.

Reply:
okay.